public/js/shopping-list-item.js

```diff
@@ -0,0 +1,21 @@
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check = function() {
+    this.isDone = true;
+  };
+
+  uncheck = function() {
+    this.isDone = false;
+  };
+
+  render = function() {
+    return `<li class="completed_${this.isDone}"><span>${this.name}</span><br><span>${
```

Comment:
Since this is a string literal, using newlines is perfectly fine here in terms of making this more readable.